examples/high_frequency/emc/double_pulse_test.py

```diff
@@ -0,0 +1,382 @@
+        [r_load.pins[1].location[0], amm_ind.pins[1].location[1]],
+        [amm_ind.pins[1].location[0], amm_ind.pins[1].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [amm_ind.pins[0].location[0], amm_ind.pins[0].location[1]],
+        [amm_bot.pins[0].location[0], amm_ind.pins[0].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [voltm_g.pins[0].location[0], voltm_g.pins[0].location[1]],
+        [v_pwl.pins[0].location[0], voltm_g.pins[0].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [v_dc_bus.pins[0].location[0], v_dc_bus.pins[0].location[1]],
+        [v_dc_bus.pins[0].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [c_dc_link.pins[0].location[0], c_dc_link.pins[0].location[1]],
+        [c_dc_link.pins[0].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [r_dc_link.pins[1].location[0], r_dc_link.pins[1].location[1]],
+        [r_dc_link.pins[1].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [nmos_l.pins[2].location[0], nmos_l.pins[2].location[1]],
+        [nmos_l.pins[2].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [voltm_ds.pins[1].location[0], voltm_ds.pins[1].location[1]],
+        [voltm_ds.pins[1].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [nmos_l.pins[2].location[0], nmos_l.pins[2].location[1]],
+        [nmos_l.pins[2].location[0], y_lower_pin],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [nmos_h.pins[2].location[0], nmos_h.pins[2].location[1]],
+        [amm_bot.pins[0].location[0], amm_bot.pins[0].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [nmos_l.pins[0].location[0], nmos_l.pins[0].location[1]],
+        [amm_bot.pins[1].location[0], amm_bot.pins[1].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [nmos_l.pins[0].location[0], nmos_l.pins[0].location[1]],
+        [voltm_ds.pins[0].location[0], voltm_ds.pins[0].location[1]],
+    ]
+)
+circuit.modeler.schematic.create_wire(
+    [
+        [v_dc_bus.pins[1].location[0], y_lower_pin],
+        [voltm_ds.pins[1].location[0], y_lower_pin],
+    ]
+)
+gnd = circuit.modeler.components.create_gnd(
+    location=[voltm_ds.pins[1].location[0], y_lower_pin - 100]
+)
+r_g1.pins[1].connect_to_component(assignment=nmos_h.pins[1], use_wire=True)
+r_g2.pins[1].connect_to_component(assignment=nmos_l.pins[1], use_wire=True)
+
+# ## Create a transient setup
+#
+
+setup_name = "MyTransient"
+setup1 = circuit.create_setup(
+    name=setup_name, setup_type=circuit.SETUPS.NexximTransient
+)
+setup1.props["TransientData"] = ["0.05ns", "15us"]
+circuit.modeler.zoom_to_fit()
+
+# Solve transient setup
+
+circuit.analyze_setup(setup_name)
```

Comment:
@tizianrot if you notice you don't use NUMCORES. This variable is always used in the analyze method. 
Here I'd suggest to use the analyze method, accessible through the setup object, so you can show how to setup the number of cores.
Please look at all the other examples.